rfcs/supported/assertion_handler/README.md

```diff
@@ -90,6 +92,10 @@ be enabled: `set_assertion_handler` and `get_assertion_handler` will be declared
 value before including oneTBB headers will disable the extension: these declarations will be excluded from
 the public API, and the library will always use the default assertion behavior.
 
+The availability of the extension can be checked with `TBB_EXT_CUSTOM_ASSERTION_HANDLER` macro. The value of it
+will be increased after adding of incompatibe changes. The macro is available after inclusion of
+`oneapi/tbb/version.h` header.
```

Comment:
I would not limit version increases to incompatible changes. Any modification of the API might deserve an increase of the macro value, so that the code can check if it can use these modifications.

Also, if we follow the C++ model for feature test macros, it should be available via both the feature header and the version header. Sorry if that was not clear in my previous comment.